File: Modulus 1/C/PRAK110-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>
#include <math.h>

int main()
{
    int alas = 5.0;
    int tinggi = 12.0;
    int sisi_miring = (alas*alas) + (tinggi*tinggi);
    int hasil_sisi_miring = sqrt(sisi_miring);
    int keliling_segitiga = alas + tinggi + hasil_sisi_miring;
    int luas = alas * tinggi /2;
    printf("diketahui : \n");
    printf("Alas = %dcm\n", alas);
    printf("Tinggi = %dcm\n\n", tinggi);
    printf("Jawab: \n");
    printf("Sisi A = %dcm\n", tinggi);
    printf("Sisi B = %dcm\n", hasil_sisi_miring);
    printf("Sisi C = %dcm\n", alas);
    printf("keliling = %dcm\n", keliling_segitiga);
    printf("Luas = %dcm\n", luas);
}

File: Modulus 1/Python/PRAK110-2310817310005-MuhammadNurwahyudiAdhitama.py
import math
alas = 12
tinggi =5
miring = round(math.sqrt(alas*alas + tinggi*tinggi))
c = round(alas+tinggi+miring)
d = round(alas*tinggi/2)
print("diketahui : ")
print(f"alas = {alas}cm")
print(f"tinggi = {tinggi}cm\n")
print(f"jawab :")
print(f"Sisi a = {tinggi}cm")
print(f"Sisi b = {miring}cm")
print(f"Sisi c = {alas}cm")
print(f"Keliling = {c}cm")
print(f"Luas = {d}cm")